packages/create-discord-bot/template/Deno/src/index.ts

```diff
@@ -1,18 +1,18 @@
 import 'https://deno.land/std@0.223.0/dotenv/load.ts';
 import { URL } from 'node:url';
 import { Client, GatewayIntentBits } from 'npm:discord.js@^14.19.3';
-import { loadCommands, loadEvents } from './util/loaders.ts';
-import { registerEvents } from './util/registerEvents.ts';
+import { loadEvents } from './util/loaders.ts';
 
 // Initialize the client
 const client = new Client({ intents: [GatewayIntentBits.Guilds] });
 
 // Load the events and commands
 const events = await loadEvents(new URL('events/', import.meta.url));
-const commands = await loadCommands(new URL('commands/', import.meta.url));
 
 // Register the event handlers
-registerEvents(commands, events, client);
+for (const event of events) {
+	client[event.once ? 'once' : 'on'](event.name, async (...args) => event.execute(...args));
```

Comment:
Why don't we do another good practice thing and wrap the event.execute in a try-catch?